### DISCUSSION TITLE
No available node version satisfies 'v22'

### DISCUSSION THREAD

**Discussion by lacozy** [yao-pkg/pkg#60] [Q&A] [answered]
PS C:\Users\lacozy\OneDrive\Documents\GitHub\TikTok-Live-Connector> node -v
v22.1.0
PS C:\Users\lacozy\OneDrive\Documents\GitHub\TikTok-Live-Connector> pkg .
> pkg@5.11.5
(node:31488) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
(Use `node --trace-deprecation ...` to show where the warning was created)
> Error! No available node version satisfies 'v22'

Was wondering why I keep experiencing this error. Thank you very much

**Comment by robertsLando** [comment, marked as answer]
Because we still don't have a nodejs 22 patch
